### DISCUSSION TITLE
Has anyone see the "value is not a sequence" error?

### DISCUSSION THREAD

**Discussion by anthonywebb** [MasterKale/SimpleWebAuthn#378] [Troubleshooting]
I am getting some users who report a "value is not a sequence" error when they try to auth on my website.  I can't find any information about this error.  Has anyone else seen it before?  I am using "@simplewebauthn/server": "^6.2.2"

**Comment by MasterKale** [comment]
Only _some_ users? Interesting...it sounds vaguely familiar, like I've come across it before, but nothing in recent memory.

Do you have any error logging you can share? And what about affected authenticators, and the browsers your affected users were using?
